annotations/kubernetes-annotations/src/main/java/io/ap4k/kubernetes/annotation/KubernetesApplication.java

```diff
@@ -32,7 +33,7 @@
 @Buildable(builderPackage = "io.ap4k.deps.kubernetes.api.builder")
 @Pojo(name = "KubernetesConfig", relativePath = "../config",
       mutable = true,
-      superClass = ApplicationConfiguration.class,
+      superClass = BaseConfig.class,
```

Comment:
I don't fully grok this, but are you sure it's correct?

Reply:
What troubles you? Maybe the name is not as descriptive as it should?

ApplicationConfiguration is the parent of all top level config classes.
BaseConfig is the base for all kubernetes classes (KubernetesConfig, OpenshiftConfig).

Reply:
Got it, makes sense now!